frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature

```diff
@@ -0,0 +1,157 @@
+Feature: UploadJAR file
+              As a user, I want to upload a JAR file from Add options
+
+        Background:
+            Given user is at developer perspective
+              And user is at Add page
+              And user has created or selected namespace "aut-upload-jar-file"
+
+
+        @regression @manual
+        Scenario Outline: Upload JAR file from Add options
+            Given user is at the Add page
+             When user clicks on the Upload JAR file card
+              And user clicks on Browse in JAR file section
+              And user selects file to upload
+              And user selects appropriate Build image version
+              And user gives Application name as "sample-upload-app" and Name as "<name>"
```

Comment:
```suggestion
              And user gives Application Groupings name as "sample-upload-app" and workload Name as "<name>"
```

Reply:
It makes more sense and won't confuse anyone. WDYT?

Reply:
I agree